microprofile/messaging/health/src/test/java/io/helidon/microprofile/messaging/health/MessagingHealthTest.java

```diff
@@ -82,16 +83,14 @@
 public class MessagingHealthTest {
 
     private static final String ERROR_MESSAGE = "BOOM!";
-
-    private WebClient client;
+    private Http1Client client;
 
     @BeforeEach
     void setUp() {
         ServerCdiExtension server = CDI.current().select(ServerCdiExtension.class).get();
         client = WebClient.builder()
-                .baseUri("http://localhost:" + server.port())
-                .addReader(JsonpSupport.reader())
-                .build();
+                 .baseUri("http://localhost:" + server.port())
```

Comment:
Since this is in MP land and the test uses `@HelidonTest` can you see if you can use the JAXRS client instead  ? You can inject a `WebTarget in the test`.